tracking_manager/models/ir_model.py

```diff
@@ -100,21 +99,27 @@ def _compute_automatic_custom_tracking(self):
     def _default_automatic_custom_tracking_domain_rules(self):
         return {
             "product.product": [
+                ("readonly", "=", False),
                 "|",
                 ("ttype", "!=", "one2many"),
                 ("name", "in", ["barcode_ids"]),
             ],
             "sale.order": [
+                ("readonly", "=", False),
                 "|",
                 ("ttype", "!=", "one2many"),
                 ("name", "in", ["order_line"]),
             ],
             "account.move": [
+                ("readonly", "=", False),
                 "|",
                 ("ttype", "!=", "one2many"),
                 ("name", "in", ["invoice_line_ids"]),
             ],
-            "default_automatic_rule": [("ttype", "!=", "one2many")],
+            "default_automatic_rule": [
+                ("ttype", "!=", "one2many"),
+                ("readonly", "=", False),
```

Comment:
```suggestion
                ("readonly", "=", True),
```
or delete it, in order to prevent readonly fields from automatic tracking.

Reply:
Are you sure? This should prevent adding readonly fields by default.
![image](https://github.com/OCA/server-tools/assets/1799080/595f9904-7daa-4908-b50d-7a2997a7bd53)
![image](https://github.com/OCA/server-tools/assets/1799080/e09433d0-a433-457f-9938-65e1925a46ce)

Reply:
My bad, you are right !